### PR TITLE
Upgrade to flask-base 0.3.1

### PR DESCRIPTION
**Pull request by nottrobin**
QA
--

`./run`

Go to:

- [x] `/`, see homepage
- [x] `/test`, see HTML page rendered from Markdown
- [x] `/non-existent`, see "404"

### CODE REVIEW

File: webapp/app.py [hunk 1]
```diff
@@ -13,18 +15,10 @@
     "certification.ubuntu.com",
     template_folder=templates_dir,
```
Comment (jpmartinspt) [inline webapp/app.py:16]: Can we remove `L9-11` and make `template_folder="templates"`?

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-canonicalwebteam.flask-base==0.1.0
+canonicalwebteam.flask-base==0.3.1
 canonicalwebteam.templatefinder==0.1.1
 Flask==1.0.2
 python-frontmatter==0.4.5
```
Comment (jpmartinspt) [inline requirements.txt]: I think this can be removed

File: webapp/app.py (hunk 1 above)
Comment (tbille) [reply webapp/app.py:16]: obviously you meant `"../templates"` :P

Comment (nottrobin) [reply webapp/app.py:16]: Actually, no unfortunately we can't until we fix https://github.com/canonical-web-and-design/canonicalwebteam.templatefinder/issues/2

(the markdown parser breaks without absolute templates path)

**Review by b-m-f — APPROVED** [review]

**Comment by b-m-f** [comment]
@nottrobin LGTM :+1:

**Pull request merged**